eslint.config.js

```diff
@@ -0,0 +1,14 @@
+import jquery from "./index.js";
+import globals from "globals";
+
+export default [
+    {
+        "languageOptions": {
+            "globals": {
+                ...globals.amd,
+                ...globals.browser
+            }
```

Comment:
These shouldn't be set globally; the configs lint themselves and only the jQuery fixture will need those globals.

Reply:
I'll put a "files" selector on this config item.